docs/index.asciidoc

```diff
@@ -356,6 +356,18 @@ This will never timeout early, but may take a little longer to timeout.
 Actual timeout is approximate based on a 250ms quantization.
 Set to 0 to disable timeouts
 
+[id="plugins-{type}s-{plugin}-timeout_scope"]
+===== `timeout_scope`
+
+  * Value type is <<string,string>>
+  * Default value is `"pattern"`
+  * Supported values are `"pattern"` and `"event"`
+
+Timeouts are per pattern matched by default but for multiple matches over a
+particular field it's usually better to scope the timeout for the whole event.
```

Comment:
```suggestion
particular field. It's usually better to scope the timeout for the whole event.
```

Reply:
does not make sense to have a dot here ... maybe we need to rephrase the sentence?
the intent here is really to explain that for multiple matches it's better to scope the timeout
> for multiple matches over a particular field it's usually better to scope ...

Reply:
I think we can resolve this by providing a little more context:

> ~~~asciidoc
> When multiple patterns are provided to <<plugins-{type}s-{plugin}-match>>,
> the timeout has historically applied to _each_ pattern, incurring overhead
> for each and every pattern that is attempted; when the grok filter is
> configured with `timeout_scope => event`, the plugin instead enforces
> a single timeout across all attempted matches on the event, so it can
> achieve similar safeguard against runaway matchers with significantly
> less overhead.
>
> It's usually better to scope the timeout for the whole event.
> ~~~

I'm not sure whether it is worth calling out that this setting has no net effect when only a single pattern is provided to `match`.